### DISCUSSION TITLE
Error: fqiled to reconnect device when exploiting checkm8

### DISCUSSION THREAD

**Discussion by ginginvv** [LukeZGD/Legacy-iOS-Kit#555] [Q&A] [answered]
I have been running into this problem from start Everything is oky until It comes to the checkm8 exploiting please help.
Im running Imac 13.6 trying to downgrade ios 10.3.4 iphone 5.

[main] CONNECTED
[main] CPID: 0x8950, BDID: 0x02, STRG: [iBoot-1145.3]
[main] Making directory: image3/
[dl_file] Downloading image: image3/ibss.n42 ...
** exploiting with checkm8
[checkm8_s5l8950x] reconnecting
[io_reset] ResetDevice: 0
[io_reset] USBDeviceReEnumerate: 0
[checkm8_s5l8950x] ERROR:Failed to reconnect to device
ERROR: Unable to connect to device
[Error] Failed to enter pwnDFU mode. Please run the script again.

* Exit DFU mode first by holding the TOP and HOME buttons for about 10 seconds.
* If you get the error "No backend available" in ipwndfu, install libusb in Homebrew/MacPorts
* For more details, read the "Troubleshooting" wiki page in GitHub
* Troubleshooting links:
    - https://github.com/LukeZGD/Legacy-iOS-Kit/wiki/Troubleshooting
    - https://github.com/LukeZGD/Legacy-iOS-Kit/wiki/Pwning-Using-Another-iOS-Device
* Legacy iOS Kit v24.08.08 (b922c90)
* Platform: macos (13.6)

**Comment by LukeZGD** [comment, marked as answer]
this is looking to be a usb cable/port/hub issue. try checking your equipment and make sure that youre using a usb-a to lightning cable
